server.go

```diff
@@ -511,17 +565,25 @@ func (srv *Server) serveTCP(l net.Listener) error {
 	}
 
 	for {
-		rw, err := l.Accept()
 		srv.lock.RLock()
 		if !srv.started {
 			srv.lock.RUnlock()
 			return nil
 		}
 		srv.lock.RUnlock()
+		rw, err := l.Accept()
```

Comment:
why is this moved down?

Reply:
why is this move down ? 
=> if Accept() is returning a TCPConn, that means there is most likely an incoming Msg. It is too late to refuse it if the server is stopped - we need to process that msg.
(which means I need to invert also on the process of the TCPConn side)

Reply:
hum.. finally, to avoid too much complexity, I will move back as initial.